README.md

```diff
@@ -24,10 +24,10 @@ PhoneGap plugin for managing application assets with javascript asset maps. Incl
 
 ### downloadFile()
 
-**wizAssets.downloadFile(String URL, String filePathToBeStoredWithFilename, Function success, Function fail);**
+**wizAssets.downloadFile(String URL, String URI, Function success, Function fail);**
```

Comment:
Okay, cool. I think it's important to document that well.
If you're saying I can come up with ANY URI imaginable, and it will always find my file by that URI, I could not be more in favor of this system. It's amazing :) No APIs can ever receive a path every again then.